react/components/MultiSelect/README.md

```diff
@@ -0,0 +1,31 @@
+```js
+initialState = {
+  selected: ["Green", "Red"]
+};
+selectableList = [
+  "White",
+  "Black",
+  "Grey",
+  "Yellow",
+  "Red",
+  "Blue",
+  "Green",
+  "Brown",
+  "Pink",
+  "Orange",
+  "Purple",
+  "Dark-blue",
+  "Dark-red",
+  "Light-blue"
+];
+<MultiSelect
+  label="Colors"
+  onChange={selected => setState({ selected: [...selected] })}
```

Comment:
nao sei se entendi o porque desse `[...selected]`. é pra duplicar o array? se for, acho melhor o proprio `onChange` já retornar um `slice` do array

Reply:
(quero dizer, poderia ser só `onChange={selected => setState({ selected })}`, mas nao sei se ta assim por necessidade)

Reply:
Não sabia que dava pra fazer isso. Boa, obrigada.